database/index.js

```diff
@@ -1,11 +1,19 @@
-var mysql = require('mysql');
+const mysql = require('mysql')
 
-var connection = mysql.createConnection({
+const connection = mysql.createConnection({
   user: 'root',
-  password: '',
-  database: 'infoPlace'
-});
+  password: 'root',
```

Comment:
Was this intentional?

Reply:
Thanks for observing that.
I was having some problems to connect with mysql and maybe in some moment I changed the password value.
I'll change it again. Thanks!